cns/fsnotify/fsnotify.go

```diff
@@ -68,7 +73,16 @@ func (w *watcher) releaseAll(ctx context.Context) {
 		}
 		file.Close()
 		podInterfaceID := string(data)
-
+		// in case of stateless CNI, CNS needs to remove HNS endpoitns first
+		if strings.Contains(podInterfaceID, "stateless") && runtime.GOOS == "windows" {
```

Comment:
we have to rethink if need "stateless" check.. this can happen only if cx running 1.6 cns reads 1.5 cni async file.. this can happen only during upgrades. what's the harm in calling delete hns endpoint for those? it would be a no-op right?